tests/cross/src/test/scala/tests/pc/SelectionRangeSuite.scala

```diff
@@ -179,4 +179,20 @@ class SelectionRangeSuite extends BaseSelectionRangeSuite {
     )
   )
 
+  private val defIsSameFor2_13And3 = List(
+    """def foo(hi: Int, >>region>>b<<region<<: Int, c:Int) = ???""".stripMargin,
+    """def foo(hi: Int, >>region>>b: Int<<region<<, c:Int) = ???""".stripMargin,
+    """def foo(>>region>>hi: Int, b: Int, c:Int<<region<<) = ???""".stripMargin,
+    """>>region>>def foo(hi: Int, b: Int, c:Int) = ???<<region<<""".stripMargin
+  )
+
+  check(
+    "def".only,
+    """def foo(hi: Int, b@@: Int, c:Int) = ???""".stripMargin,
```

Comment:
```suggestion
    """object Main { def foo(hi: Int, b@@: Int, c:Int) = ??? }""".stripMargin,
```
This is a whole Scala file and it won't correctly type check if it's not a valid Scala code and having bare `def` in Scala 2 is illegal.